public/modules/custom/grants_handler/src/Element/CompensationsComposite.php

```diff
@@ -44,20 +41,53 @@ public static function getCompositeElements(array $element): array {
     $elements['amount'] = [
       '#type' => 'textfield',
       '#title' => t('Subvention amount'),
-      '#required' => TRUE,
       '#input_mask' => "'alias': 'currency', 'prefix': '', 'suffix': '€','groupSeparator': ' ','radixPoint':','",
       '#attributes' => ['class' => ['input--borderless']],
+      '#element_validate' => ['\Drupal\grants_handler\Element\CompensationsComposite::validateAmount'],
     ];
 
     return $elements;
   }
 
+  /**
+   * Validate subvention amount.
+   *
+   * The rule here is that in SOME field must have amount inserted.
+   * So this errors if no values are given.
+   *
+   * @param array $element
+   *   Element tobe validated.
+   * @param \Drupal\Core\Form\FormStateInterface $formState
+   *   Form state.
+   * @param array $form
+   *   The form.
+   */
+  public static function validateAmount(array &$element, FormStateInterface $formState, array &$form) {
+
+    $values = $formState->getValues();
+
+    $subventionNumber = count($values['subventions']['items']);
+    $zeroes = 0;
+    unset($values['subventions']['items']);
+    foreach ($values['subventions'] as $item) {
+      if(isset($item['amount'])){
+        if ($item['amount'] == '0,00€' || empty($item['amount'])) {
+          $zeroes++;
+        }
+      }
+    }
+
+    if ($zeroes === $subventionNumber) {
+      $formState->setErrorByName('subventions', t('You must insert atleast one subvention amount'));
```

Comment:
"at least"
Missing fi and sv translations

Reply:
Done.